dictionary-octopus.txt

```diff
@@ -142,6 +142,7 @@ HKLM
 hostnames
 hotfix
 hotfixes
+Howtousecustomcertificateswith
```

Comment:
This definitely isn't a word :) I think we need a different fix for this.

Reply:
Thanks for the help @steve-fenton-octopus